server-function/src/index.ts

```diff
@@ -0,0 +1,106 @@
+import path from 'path';
+import fs from 'mz/fs';
+import mimeTypes from 'mime-types';
+import ms from 'milliseconds';
+import fresh from 'fresh';
+
+// create-react-app uses only 32 bits for hashes, allow caching for a limited
+// time for lower chance of collision
+const CACHE_TIME_SEC = ms.months(1) / 1000;
+
+export async function isFile(filePath: string): Promise<fs.Stats | undefined> {
```

Comment:
Functions named `isX` usually return boolean, this is name is confusing for the functionality.